admin_guide/pruning_resources.adoc

```diff
@@ -185,6 +185,23 @@ streams. Images are non-namespaced resources. Therefore, limiting pruning to a
 particular namespace makes it impossible to calculate their current usage.
 ====
 
+It's important to know that by default the integrated registry caches blobs metadata
+to reduce the number of requests to the storage and increase the speed of processing
+the request. Pruning doesn't update the integrated registry cache. It means that images
+that will be pushed after pruning and that contain pruned layers will be broken.
+So do this after the pruning is complete. You can run to wipe the cache:
+
+----
+$ oc rollout latest dc/docker-registry
+----
+
+In case that the integrated registry uses a
+xref:../install_config/registry/extended_registry_configuration.adoc#docker-registry-configuration-reference-redis[redis cache]
+then you need to clean the database manually.
+
+If for some reason this does not work for you, then you need to turn off all caching
+in the integrated registry.
```

Comment:
unless we have a reason to expect this to not work, i would just delete this sentence.  If we do have a reason to expect this might not work, then we should be linking to instructions on how to turn off the registry caching.

(and are we suggesting turning it off permanently, or turning it off and then back on?)

Reply:
@bparees We do not have an instruction to disable the cache, but for users who can not afford to stop the registry this is the only way out. This option is not mentioned anywhere.

If the user has several registry pods that can not be stopped and if he takes care of the data consistency, then he must permanently turn off the cache. Depending on the storage, the loss in speed can be compensated. 

I believe that we should hide such an important option.

Reply:
@legionus ok, i think i misunderstood your "if this does not work for you" statement.  I read it as "if restarting the registry does not clear the cache for some reason, then try disabling the cache"

I see now what you meant is "if redeploying the registry after pruning is not acceptable to you, then you must permanently disable the cache instead".

Can you try to make that clearer?

And if we don't have instructions for how to do it, then it seems like we need to create some, or point to some, because that's going to be the next question a user asks when they read this section.

Reply:
> Can you try to make that clearer?

@bparees Sure. Forgive my poor English.